kinetic/distribution.yaml

```diff
@@ -3038,7 +3038,7 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/UTNuclearRoboticsPublic/move_base_to_manip-release.git
-      version: 1.0.6-2
+      version: 1.0.9-0
```

Comment:
You should first fix the tracks to point to the correct branch for a specific ROS distro (otherwise it uses the default branch which is wrong for at least one distro):

* https://github.com/UTNuclearRoboticsPublic/move_base_to_manip-release/blob/c247cde96f821d83635dac7a59b177efc9c8c9ee/tracks.yaml#L15
* https://github.com/UTNuclearRoboticsPublic/move_base_to_manip-release/blob/c247cde96f821d83635dac7a59b177efc9c8c9ee/tracks.yaml#L39

Then you can run `bloom-release` again.